CONTRIBUTING.md

```diff
@@ -104,12 +106,12 @@ publishing when making changes to the `main` or `alpha` branch.
 Please note that the version published will depend on your commit message structure.
 Make sure to review and follow the instructions in the [Committing](#committing) section before committing.
 
-Before the first major release we develop against an `alpha` branch which is constantly published to [NPM](https://www.npmjs.com/package/@warp-ds/elements) and [Eik]( ) using an `alpha` tag (e.g. `1.0.0-alpha.1`).
+Before the first major release we develop against an `alpha` branch which is constantly published to [NPM](https://www.npmjs.com/package/@warp-ds/elements) and [Eik]() using an `alpha` tag (e.g. `1.0.0-alpha.1`).
```

Comment:
Can you add a url to the [Eik] hyperlink? It should be https://assets.finn.no/pkg/@warp-ds/elements